src/views/swap/steps.js

```diff
@@ -135,33 +140,22 @@ const StyledStepTwo = ({ classes, swapInfo, swapResponse }) => (
       <QrCode size={300} link={swapResponse.bip21} />
     </View>
     <View className={classes.info}>
-      <p
-        style={{
-          fontSize: '30px',
-        }}
-      >
-        Send {}
+      <p className={classes.text}>
+        Send{' '}
         <b>
-          {toWholeCoins(swapResponse.expectedAmount)} {swapInfo.base}
+          {' '}
+          {toWholeCoins(swapResponse.expectedAmount)} {swapInfo.base}{' '}
         </b>{' '}
-        to:
+        <br />
+        on <b>{getCurrencyName(swapInfo.base)}</b> <br />
+        blockchain address:
       </p>
-      <p className={classes.address} id="copy-address">
+      <p id="copy-address" className={classes.address}>
         {swapResponse.address}
       </p>
-      {/* TODO: refactor how we copy */}
       <span className={classes.action} onClick={() => copyToClipBoard()}>
         Copy
       </span>
-      <p>
-        If the address does not work with your wallet:{' '}
-        <a
-          target={'_blank'}
-          href="https://litecoin-project.github.io/p2sh-convert/"
```

Comment:
@peartobear  why did you remove that from the mockup? We need it for wallet compatibility.

Reply:
I thought about putting it in the banner UI but I think we should just put it back here for now

Reply:
@peartobear I am still not exactly sure what kind of information we should show in the UI banners or snackbars to not confuse the user. But this is certainly something that should always be available and easy to find

Reply:
@michael1011 @peartobear I followed this design https://app.zeplin.io/project/5bf801004f961c3f22c73d24/screen/5c308b0808e761af2e35498f

@peartobear I think it should always be easily available to the user.

Reply:
@ImmanuelSegol yeah I know but @peartobear shouldn't have removed it in the first place. Once you add this back in this PR can be merged.